integration/Main.purs

```diff
@@ -172,26 +179,40 @@ type ModuleResult =
   { path :: FilePath
   , errors :: Array PositionedError
   , duration :: Milliseconds
+  , mbModule :: Maybe (Module Void)
   , printerMatches :: Maybe Boolean
   }
 
 parseModuleFromFile :: FilePath -> Aff ModuleResult
 parseModuleFromFile path = do
   contents <- readTextFile UTF8 path
   before <- liftEffect hrtime
-  let parsed = parse (lex contents)
+  let parsed = parseModule contents
```

Comment:
I swapped this over to `PureScript.CST.parseModule` as that's what people will probably be using in the real world.